style.css

```diff
@@ -88,3 +97,310 @@ button {
+  flex-direction: row;
+  align-items: center;
+  color: #344563;
+  font-size: 13px;
+  font-weight: 600;
+  padding-top: -0.4em;
+}
+
+.second-small-text {
+  display: flex;
+  color: #344563;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  padding-top: 0;
+  width: 100%;
+}
+
+.dott {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  color: #7a869a;
+  font-size: 13px;
+  font-weight: 500;
+  line-height: 16px;
+}
+
+.second-text ul li:nth-child(2) {
+  margin-left: 30px;
+}
+
+p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.see-project-button {
+  color: #6070ff;
+  background: transparent;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  cursor: pointer;
+}
+
+.see-project-button:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.see-project-button:active {
+  background-color: #2230d2;
+}
+
+.small {
+  width: 295px;
+  height: 76px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.tags {
+  margin-bottom: 20px;
+  list-style: none;
+  padding-left: 0;
+}
+
+.tag {
+  float: left;
+  text-align: center;
+  margin-right: 20px;
+  padding-top: 5px;
+  align-items: center;
+  width: 53px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.javascript {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 8px;
+  width: 85px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.about-container {
+  display: grid;
+  grid-template-rows: auto auto;
+  background-color: #fff;
+  border-top-right-radius: 80px;
+  padding: 114px 24px;
+}
+
+.about-top {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+h2 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.about-text {
+  display: flex;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin-top: -30px;
+}
+
+.language-title {
+  flex: 1;
+  height: 24px;
+  margin: 15px 0 20px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.frame-lang {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 5px 0;
+  height: 66.72px;
+  margin: 5px 0 7px 0;
+  width: 100%;
+  background: #ececec;
+  border-radius: 8px;
+  padding-left: 15px;
+}
```

Comment:
- [OPTIONAL] Kindly make sure to use the same background color as the provided Figma design. You can use this color `#F7F7F9`

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/236643735-0ae386cd-0afc-4c7a-a87a-2435abbc83b7.png)

   - ### FIGMA DESIGN
   
      ![image](https://user-images.githubusercontent.com/87197412/236643722-11f019bf-13d1-41ea-8570-18e33066e7b0.png)